pkg/loop/config.go

```diff
@@ -0,0 +1,42 @@
+package loop
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+)
+
+const (
+	envLogSQLMigrations = "CL_LOG_SQL_MIGRATIONS"
+	envPromPort         = "CL_PROMETHEUS_PORT"
+)
+
+// EnvConfig is the configuration between the application and the LOOP executable. The values
+// are fully resolved and static and passed via the environment.
+type EnvConfig struct {
```

Comment:
Something felt off about the old API, and I think it's because this type was only an interface because it was capturing existing config types previous (for log fields), but that is no longer the case. A struct keeps things much simpler and more extensible (we can add fields without breaking users).

Reply:
Also allowed for simplifying the helpers to methods: `(*EnvConfig).SetCmdEnv`